src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier color = new ColorSupplier();
+        ball.setColor(color.getRandomColor());
+        Random random = new Random();
+        ball.setNumber(random.nextInt(100));
```

Comment:
lets make number 100 as constant https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names

Reply:
It's better to make number 100 as constant [mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)